### DISCUSSION TITLE
Anything within three words

### DISCUSSION THREAD

**Discussion by ploychanok** [NAIST-SE/AttractingContributors#419] [General]
Please helps me make a list of anything within three words.
It can be in any language. 
For example, ประยุทธ์ ออก ไป, にんじん が 嫌い, I love dog.

Thank you :) 

https://github.com/ploychanok/AttractingContributors-WithinThreeWords

**Comment by aru1702** [comment]
Three words sentence, this is somehow challenging

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407

**Reply by ploychanok** [reply]
Sure 👍 and Thank you so much for helping mine.
